### DISCUSSION TITLE
Minecraft Bedrock Fullbright TEXURE PACK

### DISCUSSION THREAD

**Discussion by ghost** [BedrockTweaks/Files#91] [Ideas]
I wondered, why is there no fullbright for Bedrock. I know Adonis, but these I know do not work or I have to disable Achievement and I don't want to do this

**Comment by drav0011** [comment]
We are tracking this here
https://github.com/BedrockTweaks/Files/issues/33
Not sure if it will be possible in the actual state of the game tho

**Reply by ghost** [reply]
What does that mean?

**Reply by SquatchHunter** [reply]
We cannot confirm if its even possible to make a PURELY RP version of full bright, most packs will likely be an addon, and thus disable achievements

**Reply by ghost** [reply]
But why does it then work on Java with Achievements?

**Reply by SquatchHunter** [reply]
Java is an entirely seperate game, and has a much more mature development eco system.
also Java's "Advancements" dont have to have any kind of security to them since they are set per world.
